src/promptflow-azure/promptflow/azure/_storage/blob/client.py

```diff
@@ -0,0 +1,109 @@
+import datetime
+import logging
+import threading
+import traceback
+from typing import Optional, Tuple
+
+from azure.ai.ml import MLClient
+from azure.ai.ml._azure_environments import _get_storage_endpoint_from_metadata
+from azure.ai.ml._restclient.v2022_10_01.models import DatastoreType
+from azure.ai.ml.constants._common import STORAGE_ACCOUNT_URLS
+from azure.ai.ml.entities._datastore.datastore import Datastore
+from azure.storage.blob import ContainerClient
+
+from promptflow.exceptions import UserErrorException
+
+datastore_map = {}
+client_map = {}
+_thread_lock = threading.Lock()
+_cache_timeout = 60 * 5  # Will try to refresh default datastore if exceed 5 minutes
+
+
+def get_client(
```

Comment:
Should we specify the client type more clearly? It appears that by default, this function returns a `ContainerClient` for the default blob container.

Perhaps we could rename it to `get_datastore_container_client` for clarity?

Reply:
Fixed. Thanks.